stock_getter/backend_stockgetter/stock_service/forecast_tools/tests_forecast_tools/test_ProphetModel.py

```diff
@@ -0,0 +1,97 @@
+###################################################################
+# File for testing Forecaster class
+###################################################################
+import unittest
+from ..forecaster import ProphetModel
+import pandas as pd
+
+
+###################################################################
+# Class implementing Forecaster tests
+###################################################################
+class TestProphetModel(unittest.TestCase):
+    TURN_OFF_ALL_TESTS = False
+
+    def setUp(self) -> None:
+        """This runs before every test and sets up needed objects
+        and artifacts for testing
+        """
+        # Load csv file with stock data for testing
+        fp = './backend_stockgetter/stock_service/forecast_tools'
+        fp += '/tests_forecast_tools/test_data/aapl_stk.csv'
+        self.stock_data = pd.read_csv(fp, parse_dates=['date'])
+        self.grain = 'daily'
+        self.horizon = 30
+        print('Starting tests for ProphetModel object')
+
+    def tearDown(self) -> None:
+        """Terdown function
+        for now, we just call the super method. No custom logic
+        here"""
+        return super().tearDown()
+
+
+###################################################################
+# Unit tests
+###################################################################
+    def test_run_forecast(self):
+        """Test run_forecast function
+        """
+        # Instantiate Prophet model object and run forecast
+        forecast_model = ProphetModel(self.stock_data, self.grain)
+        fcast_df = forecast_model.run_forecast(self.horizon)
+
+        # Run checks here
+        msg = "The return object should be a pandas data frame"
```

Comment:
Is msg used anywhere?

Reply:
Good catch. I need to put messages into the checks. I will push a fix.